### PR TITLE
Upgrading to Helm 3.2.0

### PR DESCRIPTION
**Pull request by swade1987**
This upgrades to using Helm 3.2.0 as Helm 3.1.2 currently contains a CVE.
See https://github.com/helm/helm/releases/tag/v3.2.0

### CODE REVIEW

**Comment by stefanprodan** [comment]
@swade1987 you need to run `go mod tidy` and commit the go.sum changes.

**Comment by swade1987** [comment]
@stefanprodan I did beforehand and it said "no changes"

**Comment by stefanprodan** [comment]
You need to bump the version in the replace section as well, then run tidy.

File: go.mod [hunk 1]
```diff
@@ -21,21 +21,21 @@ require (
...
 	github.com/yvasiyarov/go-metrics v0.0.0-20150112132944-c25f46c4b940 // indirect
 	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20160601141957-9c099fbc30e9 // indirect
 	google.golang.org/grpc v1.27.0
-	helm.sh/helm/v3 v3.1.2
-	k8s.io/api v0.17.2
-	k8s.io/apiextensions-apiserver v0.17.2
-	k8s.io/apimachinery v0.17.2
-	k8s.io/cli-runtime v0.17.2
+	helm.sh/helm/v3 v3.2.0
+	k8s.io/api v0.18.0
```
Comment (stefanprodan) [inline go.mod:29]: Please undo all changes to the Kubernetes packages as this breaks the build. Update only Helm and run go mod tidy.

**Review by stefanprodan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**